src/main/webapp/dev/js/common/instructor.js

```diff
@@ -302,6 +302,18 @@ function bindStudentPhotoHoverLink(elements) {
     });
 }
 
+/**
+ * Returns the HTML required to render the items as an unordered list.
+ */
+function generateUnorderedListHtml(items) {
+    return `\
+<ul>
+${items
+            .map(item => `<li>${item}</li>`)
+            .join('')}
```

Comment:
Is this indentation correct?

Reply:
Yes, this is the default indentation. Four/eight indentation will show lint errors.